src/Fantomas.Core/SyntaxOak.fs

```diff
@@ -32,16 +32,23 @@ type NodeBase(range: range) =
     let nodesBefore = Queue<TriviaNode>(0)
     let nodesAfter = Queue<TriviaNode>(0)
 
+    member _.ContentBefore: TriviaNode seq = nodesBefore
```

Comment:
@josh-degraw is there any practical use for this?
`NodeBase` should never be used anywhere directly.

Reply:
This way you can access those members of any node without explicitly casting to `Node`

Reply:
Sweet!